server/resolvers/article/queries.js

```diff
@@ -2,23 +2,38 @@ import { AVERAGE_WORDS_PER_MINUTE } from '../../utils/constants'
 import { databaseApi, searchApi } from '../../endpoints'
 
 const articleQueries = {
-  articles: async (_, { search }) => {
+  articles: async (_, { search }, { cache }) => {
```

Comment:
You'll probably want a test that mocks this cache out with an in-memory cache just for testing. Then you can test what's in the in-memory cache at the end of the test to make sure that things are being added correctly.

Reply:
Oh okay, so an object to mock the cache should do it right? @anniefu 

Something like `const cache = new Map()` which I can use as a local cache. The cool thing about this DS in Javascript is that it supports ops such as `set` and `get` which are essentially the same that **redis** uses to store `key/value` pairs.